src/cljs/nr/gameboard/player-stats.cljs

```diff
@@ -0,0 +1,127 @@
+(ns nr.gameboard.player-stats
+  (:require [clojure.string :as s :refer [capitalize join lower-case]]
+            [nr.appstate :refer [app-state]]
+            [nr.avatar :refer [avatar]]
+            [nr.gameboard.actions :refer [send-command]]
+            [nr.gameboard.utils :refer [inc-dec-overlay]]
+            [nr.gameboard.state :refer [game-state]]
+            [nr.translations :refer [tr tr-pronouns]]))
+
+(defn- inc-dec-overlay-for-side
+  [side]
+  (if (= (:side @game-state) side)
+    inc-dec-overlay
+    (fn [key content] content)))
+
+(defn- name-area
+  [user]
+  [:div.namearea [avatar user {:opts {:size 32}}]
```

Comment:
```suggestion
  [:div.name-area [avatar user {:opts {:size 32}}]
```
Looks like this is a holdover from the previous version of the code. Let's clean it up while we're here.